ballerina/xml_api.bal

```diff
@@ -428,3 +461,31 @@ isolated function addNamespaces(map<string> allNamespaces, map<string> namespace
         allNamespaces['key] = namespace;
     }
 }
+
+# Validates an XML document against a provided XML schema.
+#
+# The schema can either be a content of a XSD (as a `string`) or a Ballerina record type that represents
+# the XSD structure. The function checks if the `xmlValue` conforms to the provided schema.
+#
+# + schema - A `string` representing the XSD content or a Ballerina record type representing the XSD.
+# + xmlValue - The XML document that needs to be validated against the schema.
+# + return - Returns `()` if the XML is valid according to the schema, otherwise returns `Error`.
+#
+# # Examples
```

Comment:
This is not how/where we usually add examples. Fix similar to stdlib or langlib.

Reply:
Updated